New pomelo logger and fix for NetEase#1149

alexandreab opened the pull request:
Fixes NetEase#1149 

Apply changes from the branch topfreegames/pomelo _new-pomelo-logger_ to _master_

Henrod commented:
Nice. Could you just review your commits, please? There are 4 commits that are not yours. You may need to rebase with master and add just your two commits: the constructor fix and the pomelo-new-logger upgrade.

alexandreab commented:
> Nice. Could you just review your commits, please? There are 4 commits that are not yours. You may need to rebase with master and add just your two commits: the constructor fix and the pomelo-new-logger upgrade.

@Henrod Of course! I created the branch based on new-pomelo-logger branch from `topfreegames/pomelo` repository.
I just rebased with master and squashed the 4 commits in a single one. There is no change in the code.
